frescobaldi_app/language_names/generate.py

```diff
@@ -89,7 +89,7 @@ def write_dict(langs):
     with codecs.open("data.py", "w", "utf-8") as output:
         output.write("# -*- coding: utf-8;\n\n")
         output.write("# Do not edit, this file is generated. See generate.py.\n")
-        output.write("\nfrom __future__ import unicode_literals\n")
+        output.write("\n
```

Comment:
This is wrong, there is no closing quote and parenthesis.

Reply:
Thanks for spotting this. Of course it's a good thing that Frescobaldi doesn't bother to even *touch* code that isn't used yet but this of course makes it easier to overlook such errors, especially when introduced by scripts like `sed`.

Using a linter as default check is a good idea and good practice. But I'm afraid that this is a project that is out of scope given the current development resources (especially with the main author being so much out of time). I just ran `pylint` without any further configuration on both `frescobaldi_app` and `python_ly`, and the results are pretty abysmal :-(. Since I have no experience with it and don't know what should have been configured the results shouldn't be considered substantiated (an initial glance reveals that *many* warnings are about things done deliberately and should be suppressed), but rating of 5.28/10 and -0.21/10 don't look too good.
The problem is that linting wouldn't be sufficient without proper automatic testing (as you suggest), which again would require detailing the linting conditions in the contributors guide. 

Anyway, I'll open a new issue for that.

Reply:
(Fixed in https://github.com/wbsoft/frescobaldi/commit/add5af108eb9e794479113922edea99d12ffcc19, doesn't show up here automatically because the PR is merged)